packages/components/psammead-media-player/src/Canonical/index.jsx

```diff
@@ -1,38 +1,63 @@
-import React from 'react';
+import React, { useState, useRef } from 'react';
 import { string } from 'prop-types';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
+import Guidance from '../Guidance';
+import useTimeout from './useTimeout';
 
-const Canonical = ({ src, title, placeholderSrc }) => {
-  const backgroundStyle = `
-    background-image: url(${placeholderSrc});
-    background-repeat: no-repeat;
-    background-size: contain;
-  `;
+const TIMEOUT_MS = 5000;
 
-  const StyledIframe = styled.iframe`
+const Canonical = ({ service, src, title, placeholderSrc }) => {
+  const [hasTimedOut, setHasTimedOut] = useState(null);
```

Comment:
Does this need an initial value of true or false?

Reply:
`null` here is the default 'in between' state, because we're waiting for the:
- iframe to timeout `true`
- iframe to load/not timeout `false`

Reply:
Awesome thanks for the explanation @simonsinclair